tiup: add instructions for the --local option (#16097)
pingcap/docs: tiup/tiup-component-cluster-template.md

- Outputs a detailed topology template that is commented with configurable parameters. To enable this option, add it to the command.
- If this option is not specified, the simple topology template is output by default.

### --local

> - Outputs a simple topology template for local cluster which can used directly, parameter `global` can be modify by user needed.

Reviewer: ```suggestion
- Outputs a simple topology template for the local cluster, which can be used directly, and the `global` parameter can be adjusted as needed.
```

Author: Thanks

> - This template will create PD service, TiDB service, TiKV service, monitoring service and Gfafana service.

Reviewer 2: ```suggestion
- This template will create PD service, TiDB service, TiKV service, monitoring service and Grafana service.
```

Author: Thanks

> - This template will create PD service, TiDB service, TiKV service, monitoring service and Grafana service.

Reviewer: ```suggestion
- This template creates a PD service, a TiDB service, a TiKV service, a monitoring service, and a Grafana service.
```

Author: Thanks
